lib/checkWin.ts

```diff
@@ -64,29 +76,32 @@ const logGameOver = async (game, win, draw) => {
       };
     }
     game.logGrid();
+    // @ts-ignore
     game.log(log);
     game.send(gameOverObj);
   }
 };
 
-const checkForGameOver = async (game) => {
+const checkForGameOver = async (game: Game): Promise<boolean> => {
   const win = await checkForWin(game);
   const draw = await checkForDraw(game);
   await logGameOver(game, win, draw);
   return win || draw;
 };
 
-const waitForGameOver = async (game) => {
+const waitForGameOver = async (game: Game): Promise<void> => {
   let check = await checkForGameOver(game);
   while (!check) {
     await sleep(1000);
     check = await checkForGameOver(game);
   }
 };
 
-exports.mapGridValues = mapGridValues;
-exports.checkMappedGridForWin = checkMappedGridForWin;
-exports.checkForDraw = checkForDraw;
-exports.checkForWin = checkForWin;
-exports.checkForGameOver = checkForGameOver;
-exports.waitForGameOver = waitForGameOver;
+export {
+  mapGridValues,
+  checkMappedGridForWin,
+  checkForDraw,
+  checkForWin,
+  checkForGameOver,
+  waitForGameOver,
+};
```

Comment:
Replace this export section with export const on the functions themselves (do this for all files)

Reply:
#35